packages/tocco-ui/src/EditableValue/typeEditors/LocationEdit.js

```diff
@@ -0,0 +1,205 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Autosuggest from 'react-autosuggest'
+import uuid from 'uuid/v4'
+
+import ButtonLink from '../../ButtonLink'
+import {
+  StyledLocationEdit,
+  StyledZipInput
+} from './StyledLocationEdit'
+import IconTocco from '../../IconTocco'
+
+export const getMapsAddress = locationInput => {
+  const mapsBaseAddress = `https://www.google.com/maps/search/?api=1&query=`
```

Comment:
hide maps button if the generated link does not make sense. for example if address is empty...